src/WindowWrapper.cpp

```diff
@@ -1,22 +1,24 @@
 #include <Gosu/Gosu.hpp>
+#include <Gosu/WindowForWrapper.hpp>
 
 extern "C" {
   #include <Gosu/Window.h>
 
   // Constructor
   Gosu_Window* Gosu_Window_create(int width, int height, int fullscreen, double update_interval, int resizable)
   {
-    return reinterpret_cast<Gosu_Window*>( new Gosu::Window(width, height, fullscreen, update_interval, resizable) );
+    return reinterpret_cast<Gosu_Window*>( new Gosu::WindowForWrapper(width, height, fullscreen, update_interval, resizable) );
```

Comment:
I'm not sure why but supplementing `Gosu::Window` for `Gosu::WindowForWrapper` causes undefined reference errors in the resulting compiled libgosu.so.

```
/usr/bin/ld: ../gosu/cmake/build/libgosu.so: undefined reference to `Gosu::WindowForWrapper::set_draw(void (*)())'
/usr/bin/ld: ../gosu/cmake/build/libgosu.so: undefined reference to `Gosu::WindowForWrapper::WindowForWrapper(int, int, bool, double, bool)'
/usr/bin/ld: ../gosu/cmake/build/libgosu.so: undefined reference to `Gosu::WindowForWrapper::set_update(void (*)())'
```

Testing with: https://gist.github.com/cyberarm/ef957e30b0e5bb9302d6cbcc4dd892e9